front/components/RowCustom.tsx

```diff
@@ -0,0 +1,34 @@
+import { useColorScheme } from 'react-native';
+import { RootState, useSelector } from '../state/Store';
+import { Box, Pressable } from 'native-base';
+
+const RowCustom = (props: Parameters<typeof Box>[0] & { onPress?: () => void }) => {
```

Comment:
What is this component supposed to do? Isn't there a Row component already?

Reply:
RowCustom is used to create other rows such as the SongRow, it's there so you don't repete the code always used to make rows, thought it would be smarter